tasks/install.yml

```diff
@@ -1,8 +1,18 @@
 ---
 
-- name: Install requisite package repositories
+- include: tesseract-centos.yml
```

Comment:
I'm kind of worried about adding this here and I'm wondering if we would be better off moving tesseract to its own role. It could install the package for Debian, and do the compile for Cent. I don't think it would be too much work to move the tesseract stuff out of this role, and put it into its own.

Seems like a bit of a mixing of concerns the way it is right now, since the yml for compiling tesseract is ~100 lines, and the yml for installing crayfish is ~70 lines.

Reply:
Fine by me. I'll need someone to make the repo since I don't have write permissions.

Reply:
If you want to start it in your own GitHub, we can transfer it over afterwords. @dannylamb should be able to help with the transfer process when the role is ready...

Reply:
@seth-shaw-unlv Let me know when you're ready and I can set up the transfer.

Reply:
@dannylamb Just published the new [tesseract-ocr repository](https://github.com/seth-shaw-unlv/ansible-role-tesseract-ocr). Tested with both Debian and CentOS. Once this is moved over to Islandora-DevOps we can make a PR to update claw-playbook's requirements.yml and crayfish.yml.